Cargo.toml

```diff
@@ -77,3 +77,7 @@ test = false # no unit tests
 name = "rustup-init"
 path = "src/rustup-cli/main.rs"
 test = false # no unit tests
+
+[profile.release]
+lto = true
+codegen-units = 1
```

Comment:
This can be removed because with LTO enable it has no effect on the final file size the build will otherwise be faster if this is not set to 1

```suggestion
```

Reply:
Empirically, removing this line causes the size of the binary to increase by 644 KB (8.4%).

(This has consistently been my experience with LTO and codegen-units, so I think we need to re-examine some assumptions about LTO.  It's not a perfect fix for deoptimization from codegen-units.)

Reply:
Egad sorry for the delay here. This sounds fine by me though.